### PR TITLE
Added util/random.go for testing, also added test file for random.go

### PR DESCRIPTION
**Pull request by Jesselee2019**

### CODE REVIEW

File: db/util/random.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+var nameList = []string{"John", "Jane", "Joe", "Jill", "Jack"}
+
+// List of currency code for testing
+var currencyList = []string{"USD", "EUR", "CAD", "CNY", "JPY"}
+
+// By default, the random number generate number between [1, 1000]
+func GetRandomInt() int64 {
+	return rand.Int63n(1000) + 1
+}
+
```
Comment (Eyasluna) [inline db/util/random.go:17]:
``` suggestion

```

File: db/util/random.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"math/rand"
+)
+
+// List of owners names for testing
+var nameList = []string{"John", "Jane", "Joe", "Jill", "Jack"}
```
Comment (Eyasluna) [inline db/util/random.go:8]: Generate a random length of string should be fine in this case. 
``` suggestion
const alphabet = "abcdefghijklmnopqrstuvwxyz"
```

File: db/util/random.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+func GetRandomInt() int64 {
+	return rand.Int63n(1000) + 1
+}
+
+func GetRandomIntWithRange(min, max int64) int64 {
+	return rand.Int63n(max-min) + min
+}
+
+func GetRandomStringWithLength(length int64) string {
+	var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
```
Comment (Eyasluna) [inline db/util/random.go:23]: Apply line 8 into this line. `letters` should be `const` type as it should not modify.

File: db/util/random.go [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+)
+
+// List of owners names for testing
+var nameList = []string{"John", "Jane", "Joe", "Jill", "Jack"}
+
+// List of currency code for testing
+var currencyList = []string{"USD", "EUR", "CAD", "CNY", "JPY"}
+
+// By default, the random number generate number between [1, 1000]
+func GetRandomInt() int64 {
```
Comment (Eyasluna) [inline db/util/random.go:14]: We may call it `RandomTransfer` or `RandomMoney` to be easy to understand.

**Review by Eyasluna — CHANGES_REQUESTED** [review]
A few changes need to make

File: db/util/random.go [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+	b := make([]rune, length)
+	for i := range b {
+		b[i] = letters[rand.Intn(len(letters))]
+	}
+
+	return string(b)
+}
+
```
Comment (Eyasluna) [inline db/util/random.go:32]: Add a functioning comment will be better for all newly added methods. Format: `method name` + `comment`  Like this: 
``` suggestion
// GetRandomOwnerName generates a random owner name
```

File: db/util/random.go (hunk 2 above)
Comment (Jesselee2019) [reply db/util/random.go:8]: What's the rule to generate the owner's name if we are going to use `GetRandomStringWithLength()`

Comment (Eyasluna) [reply db/util/random.go:8]: I would say just use `GetRandomStringWithLength` as the method for generating a random user name. I should clear that up and I updated in ticket.

Comment (Jesselee2019) [reply db/util/random.go:8]: Sounds good, left comment on the ticket.

**Review by Eyasluna — APPROVED** [review]
LGTM

**Pull request merged**